frontend/src/components/common/Stepper.tsx

```diff
@@ -15,18 +14,18 @@ interface StepperProps {
   steps: Step[];
   activeStep: number;
   activeStepMessage?: string;
+  onChange: (step: Step, index?: number) => void;
 }
 
 /**
  * A simple stepper component that displays all based stepper components in order.
  * the active step is controlled externally via activeStep param.
  * @param param0 StepperProps
  */
-const Stepper = ({ steps, activeStep, activeStepMessage }: StepperProps) => {
-  const history = useHistory();
+const Stepper = ({ steps, activeStep, activeStepMessage, onChange }: StepperProps) => {
   return (
     <div className="bs-stepper">
-      <h6>{activeStepMessage}</h6>
+      {!!activeStepMessage && <h6>{activeStepMessage}</h6>}
       <div className="bs-stepper-header">
         {steps.map((step, index) => (
           <Fragment key={`stepper-${index}`}>
```

Comment:
I'm not sure I understand how a fragment has a key.  I guess in the virtual DOM it would exist, but in the DOM it won't?

Reply:
This appears to be supported: https://reactjs.org/docs/fragments.html#keyed-fragments